src/main/kotlin/no/nav/familie/ks/sak/kjerne/vilkårsvurdering/domene/Vilkår.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.familie.ks.sak.kjerne.vilkårsvurdering.domene
+
+import no.nav.familie.ks.sak.kjerne.beregning.domene.YtelseType
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.PersonType
+
+enum class Vilkår(
+    val parterDetteGjelderFor: List<PersonType>,
+    val ytelseType: YtelseType,
+    val beskrivelse: String,
+    val harRegelverk: Boolean
+) {
+    BOSATT_I_RIKET(
+        parterDetteGjelderFor = listOf(PersonType.SØKER, PersonType.BARN),
+        ytelseType = YtelseType.ORDINÆR_KONTANTSTØTTE,
+        beskrivelse = "Bosatt i riket",
+        true
```

Comment:
```suggestion
        parterDetteGjelderFor = listOf(PersonType.SØKER, PersonType.BARN),
        ytelseType = YtelseType.ORDINÆR_KONTANTSTØTTE,
        beskrivelse = "Bosatt i riket",
        harRegelverk  = true
```

Siden vi bruker named parameter for de 3 andre, så er det sikkert greit å kjøre på med det samme for harRegelverk 😛